aries-site/src/pages/components/table.mdx

```diff
@@ -199,7 +199,9 @@ is setting out to accomplish.
 
 See [Filtering](/templates/filtering) for greater detail.
 
-### Selecting single records
+### Navigation via table
+
+Clicking on a table row navigates to a page whose name matches the primary column in the row.
```

Comment:
I think the key is that clicking takes the user to a context where the most prominent heading matches the primary column text. This could either be a page or a modal. I think the example should be enhanced to show that as well, since the current example doesn't do that.